deepmd/pd/model/descriptor/se_atten.py

```
@@ -0,0 +1,1046 @@
+            The path to the stat file.
+
+        """
+        env_mat_stat = EnvMatStatSe(self)
+        if path is not None:
+            path = path / env_mat_stat.get_hash()
+        if path is None or not path.is_dir():
+            if callable(merged):
+                # only get data for once
+                sampled = merged()
+            else:
+                sampled = merged
```

Ruff

Use ternary operator `sampled = merged() if callable(merged) else merged` instead of `if`-`else`-block

(SIM108)